src/lightning_app/cli/cmd_pl_init.py

```diff
@@ -148,8 +149,17 @@ def print_pretty_report(
         text_pathname.stylize(f"link file://{path}")
         text_pathname.append(f" {padding} {help_text}", "blue")
 
-        icon = "📂 " if path.is_dir() else "📄 "
-        tree.add(Text(icon) + text_pathname)
+        icon = ""
+        try:
+            if path.is_dir():
+                icon = "📂 ".encode()
```

Comment:
Small suggestion: You could make this into a small helper function like this:

```py
def _encode_icon(icon: str) -> str:
    try:
        return "📂 ".encode()
    except UnicodeEncodeError:
         return b"  "
```
and then call it here in the print_pretty_report(). 

This would allow you to more easily test it, by calling the _encode_icon() function directly in your test.